sdk/monitor/Azure.Monitor.Query/src/MetricsClientOptions.cs

```diff
@@ -45,5 +45,11 @@ public enum ServiceVersion
             V2024_02_01 = 1,
 #pragma warning restore CA1707 // Identifiers should not contain underscores
         }
+
+        /// <summary>
+        /// Gets or sets the audience to use for authentication with Azure Active Directory (AAD). The audience is not considered when using a shared key.
```

Comment:
Use the new product name
```suggestion
        /// Gets or sets the audience to use for authentication with Microsoft Entra ID. The audience is not considered when using a shared key.
```